test/tasks/example/eth/008-transfer-l2pao/README.md

```diff
@@ -0,0 +1,3 @@
+# 008-transfer-l2pao: 
+
+Status: [DRAFT]()
```

Comment:
agreed with @blmalone we will delete the readme and validation files from the `test/tasks/example` folders in a separate PR since they are not used

Reply:
removed some in this PR but yes, will raise another PR to remove them all.

Reply:
This was addresses in this PR: https://github.com/ethereum-optimism/superchain-ops/pull/986